node/external/nodeApiResolver.go

```diff
@@ -333,6 +333,12 @@ func (nar *nodeApiResolver) GetManagedKeysCount() int {
 	return nar.managedPeersMonitor.GetManagedKeysCount()
 }
 
+// GetManagedKeys returns all keys managed by the current node when running in multikey mode
+func (nar *nodeApiResolver) GetManagedKeys() []string {
+	managedKeys := nar.managedPeersMonitor.GetManagedKeys()
+	return nar.parseKeys(managedKeys)
```

Comment:
before this return, can we sort the results?

Reply:
pushed + test